api/scpca_portal/metadata_file.py

```diff
@@ -96,6 +96,7 @@ def write_metadata_dicts(list_of_dicts: List[Dict], output_file_path: str, **kwa
         "fieldnames", utils.get_sorted_field_names(utils.get_keys_from_dicts(list_of_dicts))
     )
     kwargs["delimiter"] = kwargs.get("delimiter", common.TAB)
+    kwargs["restval"] = kwargs.get("restval", "NA")
```

Comment:
This was already in dev, you might need to pull the latest into the feature branch and remove this

Reply:
Handled.